lib/Phpactor.php

```diff
@@ -273,22 +273,16 @@ public static function boot(InputInterface $input, OutputInterface $output, stri
      * If the path is relative we need to use the current working path
      * because otherwise it will be the script path, which is wrong in the
      * context of a PHAR.
-     *
-     * @deprecated Use webmozart Path instead.
      */
     public static function normalizePath(string $path): string
     {
-        if (substr($path, 0, 1) == DIRECTORY_SEPARATOR) {
-            return $path;
-        }
-
-        return getcwd().DIRECTORY_SEPARATOR.$path;
+        return Path::makeAbsolute($path, (string) getcwd());
     }
```

Comment:
Question is do you want to remove this function now or keep it?

Reply:
if it's not used by anything it can be removed.